src/saga/vm-console/index.js

```diff
@@ -85,28 +55,22 @@ export function* downloadVmConsole (action) {
       }
 
       data = adjustVVFile({ data, options, usbFilter, isSpice })
-      fileDownload({ data, fileName: 'console.vv', mimeType: 'application/x-virt-viewer' })
+      fileDownload({ data, fileName: `console-${consoleId}.vv`, mimeType: 'application/x-virt-viewer' })
```

Comment:
If this is needed (I doubt, the file can be automatically removed by the desktop viewer app), then let's be consistent with the RDP bellow

Reply:
It was useful, while I was testing it, to see the different ids in the filename between VNC and SPICE on the same VM.  Also I'm not sure what id I'd add to the RDP file name since it doesn't have a console id.

Oh, I just thought of this ... what happens if a user has >1 VM set to auto-open its console?  I guess the browser would rename the second file or ask the user where to save it.  Adding the console id would help step around that scenario a bit more gracefully.

Reply:
I think it's same approach as Web Admin does. There's presumption, the user does not click so fast :-)

The expected flow is: download file, open associated application which removes the file. Might take a few seconds.

So if you think this is an issue, you can add unique appendix to the filename. IMO, there must not be reference to the VM. I think, value of a counter is good enough. Same for the RDP.
But as I wrote - just if you see issue here.

Reply:
Backed the change out, I don't think it is really needed.  User's browser can handle multiple file downloads if they have auto-connect turned on for >1 VM.

Reply:
:man_facepalming: I didn't realize that the "connect automatically" switch only holds on to 1 VM id in browser local storage, so it's really a moot point now.